pybedtools/test/data/issue319.out.bed

```diff
@@ -0,0 +1,3496 @@
on [(RodBinding name=variant62 source=/n/scratch2/hsph_bioinformatic_core/sn240/atanasova2020/atanasova/work/variation/rnaseq/gatk-haplotype/S67-gatk-haplotype.vcf.gz)]), (RodBindingCollection [(RodBinding name=variant63 source=/n/scratch2/hsph_bioinformatic_core/sn240/atanasova2020/atanasova/work/variation/rnaseq/gatk-haplotype/S68-gatk-haplotype.vcf.gz)]), (RodBindingCollection [(RodBinding name=variant64 source=/n/scratch2/hsph_bioinformatic_core/sn240/atanasova2020/atanasova/work/variation/rnaseq/gatk-haplotype/S69-gatk-haplotype.vcf.gz)]), (RodBindingCollection [(RodBinding name=variant65 source=/n/scratch2/hsph_bioinformatic_core/sn240/atanasova2020/atanasova/work/variation/rnaseq/gatk-haplotype/S6-gatk-haplotype.vcf.gz)]), (RodBindingCollection [(RodBinding name=variant66 source=/n/scratch2/hsph_bioinformatic_core/sn240/atanasova2020/atanasova/work/variation/rnaseq/gatk-haplotype/S70-gatk-haplotype.vcf.gz)]), (RodBindingCollection [(RodBinding name=variant67 source=/n/scratch2/hsph_bioinformatic_core/sn240/atanasova2020/atanasova/work/variation/rnaseq/gatk-haplotype/S71-gatk-haplotype.vcf.gz)]), (RodBindingCollection [(RodBinding name=variant68 source=/n/scratch2/hsph_bioinformatic_core/sn240/atanasova2020/atanasova/work/variation/rnaseq/gatk-haplotype/S72-gatk-haplotype.vcf.gz)]), (RodBindingCollection [(RodBinding name=variant69 source=/n/scratch2/hsph_bioinformatic_core/sn240/atanasova2020/atanasova/work/variation/rnaseq/gatk-haplotype/S73-gatk-haplotype.vcf.gz)]), (RodBindingCollection [(RodBinding name=variant70 source=/n/scratch2/hsph_bioinformatic_core/sn240/atanasova2020/atanasova/work/variation/rnaseq/gatk-haplotype/S74-gatk-haplotype.vcf.gz)]), (RodBindingCollection [(RodBinding name=variant71 source=/n/scratch2/hsph_bioinformatic_core/sn240/atanasova2020/atanasova/work/variation/rnaseq/gatk-haplotype/S7-gatk-haplotype.vcf.gz)]), (RodBindingCollection [(RodBinding name=variant72 source=/n/scratch2/hsph_bioinformatic_core/sn240/atanasova2020/atanasova/work/variation/rnaseq/gatk-haplotype/S8-gatk-haplotype.vcf.gz)]), (RodBindingCollection [(RodBinding name=variant73 source=/n/scratch2/hsph_bioinformatic_core/sn240/atanasova2020/atanasova/work/variation/rnaseq/gatk-haplotype/S9-gatk-haplotype.vcf.gz)])] out=/n/scratch2/hsph_bioinformatic_core/sn240/atanasova2020/atanasova/work/bcbiotx/tmp_y699b1y/b1-chr1_0_248956422.vcf.gz includeNonVariantSites=false uniquifySamples=false annotateNDA=false useNewAFCalculator=false heterozygosity=0.001 indel_heterozygosity=1.25E-4 heterozygosity_stdev=0.01 standard_min_confidence_threshold_for_calling=10.0 standard_min_confidence_threshold_for_emitting=30.0 max_alternate_alleles=4 max_genotype_count=1024 max_num_PL_values=100 input_prior=[] sample_ploidy=2 annotation=[] group=[StandardAnnotation] dbsnp=(RodBinding name=dbsnp source=/n/app/bcbio/dev/genomes/Hsapiens/hg38/variation/dbsnp-151.vcf.gz) filter_reads_with_N_cigar=false filter_mismatching_base_and_quals=false filter_bases_not_stored=false">
```

Comment:
This line should be taken out -- these test files are distributed with the pybedtools package.

Are all the headers of this file required for reproducing the test? I'd prefer to only have what's needed for a minimally-reproducible example.